go.mod

```diff
@@ -99,7 +99,7 @@ require (
 	github.com/hashicorp/vault-plugin-auth-azure v0.11.0
 	github.com/hashicorp/vault-plugin-auth-centrify v0.12.0
 	github.com/hashicorp/vault-plugin-auth-cf v0.12.0
-	github.com/hashicorp/vault-plugin-auth-gcp v0.7.1-0.20220630160554-54acedf4f4f6
+	github.com/hashicorp/vault-plugin-auth-gcp v0.13.2-0.20220722185016-9e4fddb995e5
```

Comment:
Ah, I was wondering how this went to v0.13.2. Looks like the plugin was incorrectly tagged from main in https://github.com/hashicorp/vault-plugin-auth-gcp/commit/eee8ee8ee9ce65ed04935c3511e7604ecfa0e855.

Reply:
I'm not sure if there is anything we can do about it now. Let me see.

Reply:
Not much we can do here. Resolving this.